### ISSUE TITLE
Running the example app created by `npx create-expo-module my-module` fails

### ISSUE THREAD

**Issue by iketw** [expo/expo#27455]
### Summary

Steps to reproduce:

1. Run `npx create-expo-module my-module`
2. `cd my-module`
3. `cd example`
4. npm run android

Error:
`error: node_modules/expo/AppEntry.js: Cannot find module 'babel-plugin-module-resolver'`

After I added `babel-plugin-module-resolver` the error was gone and the app ran successfully

### What platform(s) does this occur on?

Android

### SDK Version

"expo": "~50.0.8"

### Environment

expo-env-info 1.2.0 environment info:
    System:
      OS: macOS 14.0
      Shell: 5.9 - /bin/zsh
    Binaries:
      Node: 18.0.0 - ~/.nvm/versions/node/v18.0.0/bin/node
      Yarn: 1.22.19 - /opt/homebrew/bin/yarn
      npm: 8.6.0 - ~/.nvm/versions/node/v18.0.0/bin/npm
      Watchman: 2023.12.04.00 - /opt/homebrew/bin/watchman
    Managers:
      CocoaPods: 1.12.1 - /Users/dalmendraytw/.rvm/gems/ruby-2.7.6/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 23.0, iOS 17.0, macOS 14.0, tvOS 17.0, watchOS 10.0
      Android SDK:
        API Levels: 29, 30, 31, 32, 33, 33, 34
        Build Tools: 28.0.3, 29.0.2, 30.0.2, 30.0.3, 31.0.0, 32.0.0, 33.0.0, 33.0.1, 34.0.0
        System Images: android-29 | ARM 64 v8a, android-29 | Intel x86 Atom, android-29 | Intel x86 Atom_64, android-29 | Google APIs ARM 64 v8a, android-29 | Google APIs Intel x86 Atom, android-29 | Google APIs Intel x86 Atom_64, android-29 | Google Play ARM 64 v8a, android-29 | Google Play Intel x86 Atom, android-29 | Google Play Intel x86 Atom_64, android-32 | Google APIs ARM 64 v8a
        Android NDK: 24.0.8215888
    IDEs:
      Android Studio: 2022.3 AI-223.8836.35.2231.10406996
      Xcode: 15.0.1/15A507 - /usr/bin/xcodebuild
    npmPackages:
      expo: ~50.0.8 => 50.0.8 
      react: 18.2.0 => 18.2.0 
      react-native: 0.73.4 => 0.73.4 
    npmGlobalPackages:
      eas-cli: 3.8.0
    Expo Workflow: bare

### Minimal reproducible example

It's really easy to reproduce, please, just follow the steps in the Summary

**Comment by byCedric** [comment]
@tsapeta Can we release the current version of the `expo-module-template` that uses `metro.resolver.extraNodeModules` instead of `babel-plugin-moduler-resolver`?

From: https://github.com/expo/expo/pull/27017

**Comment by tsapeta** [comment]
@byCedric Yup, I think it was released but then someone else overrode that by releasing from another branch 🙈

**Comment by tsapeta** [comment]
I released a new template that should fix that issue, so I'm going to close it. Please let us know if you're still encountering it.

**Comment by iketw** [comment]
Yeap, working now. Thanks for the quick turnaround
